testsuite/tests/kuadrant/authorino/wristband/conftest.py

```diff
@@ -70,9 +70,7 @@ def wristband_endpoint(openshift, authorino, authorization_name):
 @pytest.fixture(scope="module")
 def authorization(authorization, wristband_secret, wristband_endpoint) -> AuthConfig:
     """Add wristband response with the signing key to the AuthConfig"""
-    authorization.responses.add_wristband(
-        "wristband", wristband_endpoint, wristband_secret, wrapper="envoyDynamicMetadata"
-    )
+    authorization.responses.add_wristband("wristband", wristband_endpoint, wristband_secret, wrapper="dynamicMetadata")
```

Comment:
Does the wristband always need `dynamicMetadata` or can you use it through the header as well?

Reply:
According to documentation it can be both https://github.com/Kuadrant/authorino/blob/main/docs/user-guides/deny-with-redirect-to-login.md#festival-wristband-tokens-responsesuccessheadersdynamicmetadatawristband